### ISSUE TITLE
Game crashes on start

### ISSUE THREAD

**Issue by AugoeidesGaming** [CyclopsMC/EverlastingAbilities#128]
#### Issue type:
Mod seems to crash my game.
____
#### Short description:
The game crashes upon start up.

#### Steps to reproduce the problem:
1. The game crashes after startup from the Launcher, and I get the following message:
**The game crashed whilst rendering overlay
Error: java.lang.NullPointerException: Cannot get config value without assigned Config object present**
____
#### Versions:
- This mod: Everlasting Abilities v1.14.4-1.5.1
- Minecraft: v1.14.4
- Forge: v28.0.49

#### Log file:
---- Minecraft Crash Report ----
// My bad.

Time: 9/12/19 3:24 PM
Description: Rendering overlay

java.lang.NullPointerException: Cannot get config value without assigned Config object present
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:787) ~[guava-21.0.jar:?] {}
	at net.minecraftforge.common.ForgeConfigSpec$ConfigValue.get(ForgeConfigSpec.java:682) ~[?:?] {}
	at org.cyclops.cyclopscore.config.ConfigurablePropertyData.saveToField(ConfigurablePropertyData.java:123) ~[?:1.5.0] {}
	at org.cyclops.cyclopscore.config.ConfigHandler.syncProcessedConfigs(ConfigHandler.java:186) ~[?:1.5.0] {}
	at org.cyclops.cyclopscore.config.ConfigHandler.onLoad(ConfigHandler.java:107) ~[?:1.5.0] {}
	at net.minecraftforge.eventbus.ASMEventHandler_2_ConfigHandler_onLoad_Loading.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:80) ~[eventbus-0.10.6-service.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:258) ~[eventbus-0.10.6-service.jar:?] {}
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.lambda$new$0(FMLModContainer.java:66) ~[?:28.0] {}
	at net.minecraftforge.fml.javafmlmod.FMLModContainer$$Lambda$2028/213662707.accept(Unknown Source) ~[?:?] {}
	at net.minecraftforge.fml.ModContainer.lambda$dispatchConfigEvent$5(ModContainer.java:147) ~[?:?] {}
	at net.minecraftforge.fml.ModContainer$$Lambda$3960/1585465432.accept(Unknown Source) ~[?:?] {}
	at java.util.Optional.ifPresent(Optional.java:159) ~[?:1.8.0_51] {}
	at net.minecraftforge.fml.ModContainer.dispatchConfigEvent(ModContainer.java:147) ~[?:?] {}
	at net.minecraftforge.fml.config.ModConfig.fireEvent(ModConfig.java:92) ~[?:?] {}
	at net.minecraftforge.fml.config.ConfigTracker.openConfig(ConfigTracker.java:101) ~[?:?] {}
	at net.minecraftforge.fml.config.ConfigTracker.lambda$loadConfigs$1(ConfigTracker.java:83) ~[?:?] {}
	at net.minecraftforge.fml.config.ConfigTracker$$Lambda$3947/1574820764.accept(Unknown Source) ~[?:?] {}
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:1.8.0_51] {}
	at java.util.Collections$SynchronizedCollection.forEach(Collections.java:2062) ~[?:1.8.0_51] {}
	at net.minecraftforge.fml.config.ConfigTracker.loadConfigs(ConfigTracker.java:83) ~[?:?] {}
	at net.minecraftforge.fml.ModLoader.lambda$null$5(ModLoader.java:148) ~[?:?] {}
	at net.minecraftforge.fml.ModLoader$$Lambda$3944/516266724.run(Unknown Source) ~[?:?] {}
	at net.minecraftforge.fml.DistExecutor.runWhenOn(DistExecutor.java:56) ~[?:?] {}
	at net.minecraftforge.fml.ModLoader.loadMods(ModLoader.java:148) ~[?:?] {}
	at net.minecraftforge.fml.client.ClientModLoader.lambda$startModLoading$6(ClientModLoader.java:105) ~[?:?] {pl:runtimedistcleaner:A}
	at net.minecraftforge.fml.client.ClientModLoader$$Lambda$3937/252393895.run(Unknown Source) ~[?:?] {}
	at net.minecraftforge.fml.client.ClientModLoader.lambda$createRunnableWithCatch$5(ClientModLoader.java:95) ~[?:?] {pl:runtimedistcleaner:A}
	at net.minecraftforge.fml.client.ClientModLoader$$Lambda$1953/562337324.run(Unknown Source) ~[?:?] {}
	at net.minecraftforge.fml.client.ClientModLoader.startModLoading(ClientModLoader.java:105) ~[?:?] {pl:runtimedistcleaner:A}
	at net.minecraftforge.fml.client.ClientModLoader.lambda$onreload$3(ClientModLoader.java:87) ~[?:?] {pl:runtimedistcleaner:A}
	at net.minecraftforge.fml.client.ClientModLoader$$Lambda$3936/881864678.run(Unknown Source) ~[?:?] {}
	at net.minecraftforge.fml.client.ClientModLoader.lambda$createRunnableWithCatch$5(ClientModLoader.java:95) ~[?:?] {pl:runtimedistcleaner:A}
	at net.minecraftforge.fml.client.ClientModLoader$$Lambda$1953/562337324.run(Unknown Source) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1618) ~[?:1.8.0_51] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1610) ~[?:1.8.0_51] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) ~[?:1.8.0_51] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) ~[?:1.8.0_51] {}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1689) ~[?:1.8.0_51] {}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) ~[?:1.8.0_51] {}

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:787)
	at net.minecraftforge.common.ForgeConfigSpec$ConfigValue.get(ForgeConfigSpec.java:682)
	at org.cyclops.cyclopscore.config.ConfigurablePropertyData.saveToField(ConfigurablePropertyData.java:123)
	at org.cyclops.cyclopscore.config.ConfigHandler.syncProcessedConfigs(ConfigHandler.java:186)
	at org.cyclops.cyclopscore.config.ConfigHandler.onLoad(ConfigHandler.java:107)
	at net.minecraftforge.eventbus.ASMEventHandler_2_ConfigHandler_onLoad_Loading.invoke(.dynamic)
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:80)
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:258)
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.lambda$new$0(FMLModContainer.java:66)
	at net.minecraftforge.fml.javafmlmod.FMLModContainer$$Lambda$2028/213662707.accept(Unknown Source)
	at net.minecraftforge.fml.ModContainer.lambda$dispatchConfigEvent$5(ModContainer.java:147)
	at net.minecraftforge.fml.ModContainer$$Lambda$3960/1585465432.accept(Unknown Source)
	at java.util.Optional.ifPresent(Optional.java:159)
	at net.minecraftforge.fml.ModContainer.dispatchConfigEvent(ModContainer.java:147)
	at net.minecraftforge.fml.config.ModConfig.fireEvent(ModConfig.java:92)
	at net.minecraftforge.fml.config.ConfigTracker.openConfig(ConfigTracker.java:101)
	at net.minecraftforge.fml.config.ConfigTracker.lambda$loadConfigs$1(ConfigTracker.java:83)
	at net.minecraftforge.fml.config.ConfigTracker$$Lambda$3947/1574820764.accept(Unknown Source)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at java.util.Collections$SynchronizedCollection.forEach(Collections.java:2062)
	at net.minecraftforge.fml.config.ConfigTracker.loadConfigs(ConfigTracker.java:83)
	at net.minecraftforge.fml.ModLoader.lambda$null$5(ModLoader.java:148)
	at net.minecraftforge.fml.ModLoader$$Lambda$3944/516266724.run(Unknown Source)
	at net.minecraftforge.fml.DistExecutor.runWhenOn(DistExecutor.java:56)
	at net.minecraftforge.fml.ModLoader.loadMods(ModLoader.java:148)

-- Overlay render details --
Details:
	Overlay name: net.minecraft.client.gui.ResourceLoadProgressGui
Stacktrace:
	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:544)
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:886)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:384)
	at net.minecraft.client.main.Main.main(SourceFile:155)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:56)
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$349/389247924.call(Unknown Source)
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37)
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:50)
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:68)
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:80)
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65)

-- System Details --
Details:
	Minecraft Version: 1.14.4
	Minecraft Version ID: 1.14.4
	Operating System: Windows 8.1 (amd64) version 6.3
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 7480498688 bytes (7133 MB) / 8589934592 bytes (8192 MB) up to 8589934592 bytes (8192 MB)
	CPUs: 8
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx8G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	ModLauncher: 3.2.0+60+b86c1d4
	ModLauncher launch target: fmlclient
	ModLauncher naming: srg
	ModLauncher services: 
		/eventbus-0.10.6-service.jar eventbus PLUGINSERVICE 
		/forge-1.14.4-28.0.49.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.14.4-28.0.49.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-0.16.0-shadowed.jar accesstransformer PLUGINSERVICE 
		/forge-1.14.4-28.0.49.jar capability_inject_definalize PLUGINSERVICE 
		/forge-1.14.4-28.0.49.jar runtimedistcleaner PLUGINSERVICE 
		/forge-1.14.4-28.0.49.jar fml TRANSFORMATIONSERVICE 
	FML: 28.0
	Forge: net.minecraftforge:28.0.49
	FML Language Providers: 
		javafml@28.0
		minecraft@1
		kotori_scala@2.12.8
	Mod List: 
		[1.14.2-1.14.4]TorchBowMod_v1.4.jar TorchBowMod {torchbowmod@1.4 COMMON_SETUP}
		aerialaffinity-3.0a.jar Aerial Affinity {aerialaffinity@3.0a COMMON_SETUP}
		AmbientSounds_v3.0.9_mc1.14.4.jar Ambient Sounds {ambientsounds@3.0.3 COMMON_SETUP}
		AttributeFix-1.14.4-3.0.2.jar AttributeFix {attributefix@3.0.2 COMMON_SETUP}
		bambooeverything-1.14-1.2.0.jar Bamboo Everything {bambooeverything@1.14-1.2.0 COMMON_SETUP}
		BasicNetherOres-1.14.4-3.2.0.jar Basic Nether Ores {bno@3.2.0 COMMON_SETUP}
		Batty's+Coordinates+PLUS+Mod+for+Forge-1.14.4-0.1.1.jar Batty's Coordinates Plus Mod for Forge {coordinatesplusmod@1.14.4_0.1.1 COMMON_SETUP}
		BetterAnimationsCollection2-v1.1b1-1.14.4.jar Better Animations Collection 2 {betteranimationscollection2@1.1b1 COMMON_SETUP}
		BetterStorageToo-1.14.3-4.0.0.0-alpha3.jar BetterStorageToo {betterstorage@1.14.3-4.0.0.0-alpha3 COMMON_SETUP}
		branders-enhanced-mob-spawners.v1.4.2-1.14.4.jar Enhanced Mob Spawners {spawnermod@1.4.2 COMMON_SETUP}
		camera-1.4.4.jar Camera Mod {camera@1.4.4 COMMON_SETUP}
		campfiretorches-1.14.4-1.0.2.jar Campfire Torches {campfiretorches@1.14.4-1.0.0 COMMON_SETUP}
		Charm-1.14.4-1.3.2.jar Charm {charm@1.14.4-1.3.2 COMMON_SETUP}
		Clumps-4.0.0.jar Clumps {clumps@4.0.0 COMMON_SETUP}
		CosmeticArmorReworked-1.14.4-v1a.jar CosmeticArmorReworked {cosmeticarmorreworked@1.14.4-v1a COMMON_SETUP}
		CTD-Core-1.14.4-2.0.4.jar CTD Core {ctdcore@2.0.4 COMMON_SETUP}
		CyclopsCore-1.14.4-1.5.0.jar Cyclops Core {cyclopscore@1.5.0 COMMON_SETUP}
		damage_indicator-0.0.4.jar Ama's Damage Indicator {damage_indicator@0.0.4 COMMON_SETUP}
		DoggyTalents-1.14.4-1.15.1.3-universal.jar Doggy Talents {doggytalents@1.15.1.3 COMMON_SETUP}
		DungeonCrawl-1.14.4-1.3.0.jar Dungeon Crawl {dungeoncrawl@1.3.0 COMMON_SETUP}
		dwmh-1.14.4-3.0.1.jar Dude! Where's my Horse? {dwmh@1.14.4-3.0.1 COMMON_SETUP}
		EnchantmentDescriptions-1.14.4-1.3.4.jar EnchantmentDescriptions {enchdesc@1.3.4 COMMON_SETUP}
		enchxchg-1.14.4-4.0.3.jar Enchantments Exchanger {enchxchg@1.14.4-4.0.3 COMMON_SETUP}
		Enemyz-1.14.4-0.2.3.jar Enemyz {enemyz@0.2.3 COMMON_SETUP}
		EquipmentTooltips-1.14.4-1.3.1+9.jar Equipment Tooltips {equipmenttooltips@1.3.1+9 COMMON_SETUP}
		EverlastingAbilities-1.14.4-1.5.1.jar EverlastingAbilities {everlastingabilities@1.14.4-1.5.1 COMMON_SETUP}
		extrabows-v1.14.4+b2.jar Extra Bows {extrabows@v1.14.4 b2 COMMON_SETUP}
		FastFurnace-1.14.4-2.0.0.jar FastFurnace {fastfurnace@2.0.0 COMMON_SETUP}
		FastLeafDecay-v17.jar FastLeafDecay {fastleafdecay@v17 COMMON_SETUP}
		Fence+Jumper-1.14.4-1.0.8.jar FenceJumper {fencejumper@version COMMON_SETUP}
		fishtraps-0.0.4.jar Fish Traps {fishtraps@1.14.4-0.0.4 COMMON_SETUP}
		furniture-7.0.0-pre4-1.14.4.jar MrCrayfish's Furniture Mod {cfm@7.0.0-pre4 COMMON_SETUP}
		glassential-forge-1.14.4-1.1.1.jar Glassential {glassential@1.1.1 COMMON_SETUP}
		globalxp-1.14.4-v1.4.5.jar Global XP {globalxp@v1.4.5 COMMON_SETUP}
		goodnightsleep-1.14.4-0.3.0.jar Good Night's Sleep {goodnightsleep@0.3.0 COMMON_SETUP}
		ilikewood-release-1.14.4-19w32a.jar I Like Wood {ilikewood@release-1.14.4-19w32a COMMON_SETUP}
		BiomesOPlenty-1.14.4-9.0.0.251-universal.jar Biomes O' Plenty {biomesoplenty@1.14.4-9.0.0.251 COMMON_SETUP}
		ironchest-1.14.3-9.0.DEV.55a7af2.jar Iron Chests {ironchest@1.14.3-9.0.DEV.55a7af2 COMMON_SETUP}
		jei-1.14.4-6.0.0.10.jar Just Enough Items {jei@6.0.0.10 COMMON_SETUP}
		LargeFluidTank-14.6.7.jar FluidTank {fluidtank@14.6.7 COMMON_SETUP}
		levelhearts-1.14.4-1.1.1.jar LevelHearts {levelhearts@1.1.1 COMMON_SETUP}
		MCFurnace-1.14.4-DEV-1.2.1.jar MCFurnace {mcfurnace@1.0 COMMON_SETUP}
		Mo-Glowstone-1.14.4-2.0.2.jar Mo' Glowstone {moglowstone@2.0.2 COMMON_SETUP}
		Neat+1.5-19.jar Neat {neat@1.5-19 COMMON_SETUP}
		PackingTape-1.14.4-0.8.3.jar Packing Tape {packingtape@0.8.3 COMMON_SETUP}
		Placebo-1.14.4-2.1.5.jar Placebo {placebo@2.1.5 COMMON_SETUP}
		prefab-1.4.0.2.jar Prefab {prefab@1.4.0.2 COMMON_SETUP}
		RandomThings-MC1.14.4-5.0.1.jar Random Things {randomthings@5.0.1 COMMON_SETUP}
		reap-1.10.1.jar Reap Mod {reap@1.10.1 COMMON_SETUP}
		respawnablepets-1.0.4.2.jar Respawnable Pets {respawnablepets@1.0.4.2 COMMON_SETUP}
		RingOfAttraction-1.14.4-1.0.5.jar Ring of Attraction {ring_of_attraction@1.0.5 COMMON_SETUP}
		rsgauges-1.14.4-1.1.7-b2.jar Redstone Gauges and Switches {rsgauges@1.1.7-b2 COMMON_SETUP}
		ruby-1.14.4.0.jar Just Another Ruby Mod {ruby@version COMMON_SETUP}
		ruins-1.14.4.1.jar Ruins {ruins@1.14.3.3 COMMON_SETUP}
		scarecrows-1.14.4-v1.0.8.jar Scarecrows {scarecrows@v1.0.8 COMMON_SETUP}
		ShoulderSurfing-1.14.4-1.19.jar Shoulder Surfing {shouldersurfing@1.14.2-1.19 COMMON_SETUP}
		SignButton-1.14.2-2.0.1.jar Sign Button {signbutton@version COMMON_SETUP}
		SimpleDivingGear-3.0.2+mc+1.14.4.jar Simple Diving Gear {simpledivegear@3.0.2 mc 1.14.4 COMMON_SETUP}
		simplefarming-1.14.4-1.0.6.jar Simple Farming {simplefarming@1.14.4-1.0.6 COMMON_SETUP}
		SimpleStorageNetwork-1.14.4-0.0.10.jar Simple Storage Network {storagenetwork@1.14.4-0.0.10 COMMON_SETUP}
		Simply+Platinum+-+MC+1.14.4+(1.8).jar Simply Platinum {simplyplatinum@1.8 COMMON_SETUP}
		sit-1.14.4-v1.1.3.jar Sit {sit@v1.1.3 COMMON_SETUP}
		snowvariants-1.14.4-1.0.2.jar Snow Variants {snowvariants@1.14.4-1.0.2 COMMON_SETUP}
		speedyhoppers-1.14.4-1.0.2.jar Speedy Hoppers {speedyhoppers@1.14.4-1.0.2 COMMON_SETUP}
		speedyladders-1.14.4-1.0.1.jar Speedy Ladders {speedyladders@1.14.4-1.0.1 COMMON_SETUP}
		SuperTools-1.14.4_1.3c.jar Super Tools {supertools@1.14.4_1.3c COMMON_SETUP}
		themidnight-0.4.4.jar The Midnight {midnight@0.4.4 COMMON_SETUP}
		tombstone-3.7.5-1.14.4.jar Corail Tombstone {tombstone@3.7.5 COMMON_SETUP}
		torchmaster-2.0.0.6-alpha.jar Torchmaster {torchmaster@2.0.0.6-alpha COMMON_SETUP}
		toughnessbar-4.0.jar Toughness Bar {toughnessbar@4.0 COMMON_SETUP}
		tramplestopper-1.14.4-2.0.0.22-universal.jar Trample Stopper {tramplestopper@1.14.4-2.0.0.22-universal COMMON_SETUP}
		travel_bag-1.0.2-1.14.4.jar Travel Bag {travel_bag@1.0.2 COMMON_SETUP}
		travellersboots-1.14.4-1.2.1.jar Traveller's Boots {travellersboots@1.2.1 COMMON_SETUP}
		u_team_core-1.14.4-2.7.0.129.jar U Team Core {uteamcore@2.7.0.129 COMMON_SETUP}
		useful_backpacks-1.14.4-1.7.0.34.jar Useful Backpacks {usefulbackpacks@1.7.0.34 COMMON_SETUP}
		voyage-1.0.0.jar Voyage {voyage@1.0.0 COMMON_SETUP}
		Waddles-1.14.4-0.8.3.jar Waddles {waddles@1.14.4-0.8.3 COMMON_SETUP}
		wild_netherwart-1.14.4-v3.0.jar Wild Netherwart {wildnetherwart@1.0 COMMON_SETUP}
		Xaeros_Minimap_1.17.8_Forge_1.14.4.jar Xaero's Minimap {xaerominimap@1.17.8 COMMON_SETUP}
		XaerosWorldMap_1.4.8_Forge_1.14.4.jar Xaero's World Map {xaeroworldmap@1.4.8 COMMON_SETUP}
		forge-1.14.4-28.0.49-universal.jar Forge {forge@28.0.49 COMMON_SETUP}
		forge-1.14.4-28.0.49-client.jar Minecraft {minecraft@1.14.4 COMMON_SETUP}
	Launched Version: 1.14.4-forge-28.0.49
	LWJGL: 3.2.2 build 10
	OpenGL: GeForce GTX 1060 6GB/PCIe/SSE2 GL version 4.6.0 NVIDIA 436.02, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'
	Type: Client (map_client.txt)
	Resource Packs: vanilla, file/WASD Magical Rings Resource Pack, file/WASD Magical Rods Resource Pack, file/WASD Keese Resource Pack
	Current Language: English (US)
	CPU: 8x Intel(R) Core(TM) i7-4790 CPU @ 3.60GHz

**Comment by rubensworks** [comment]
This has been fixed in more recent Forge versions, you're running quite a couple of versions behind.

**Comment by AugoeidesGaming** [comment]
> 
> 
> This has been fixed in more recent Forge versions, you're running quite a couple of versions behind.

The majority of the Minecraft modding community operates on about Forge v28.0.45, because Forge updates almost daily, so not every mod is going to be synchronized with the others.

However, I took your advice and updated my Forge anyway, testing how far I could go without losing any mods, but updating any further than Forge v28.0.58 flat out crashes my game every time, probably because of all the mods I have installed.

Even with this newer version of Forge, I still can't run Everlasting Abilities.
